bitnami/thanos/templates/_helpers.tpl

```diff
@@ -449,3 +449,22 @@ Usage:
 {{- end -}}
 {{- end -}}
 {{- end -}}
+
+{{/*
+Labels to use on serviceMonitor.spec.selector and svc.metadata.labels
+*/}}
+{{- define "thanos.servicemonitor.matchLabels" -}}
+{{- if and .Values.metrics.enabled .Values.metrics.serviceMonitor.enabled -}}
+prometheus-operator/monitor: 'true'
+{{- end }}
+{{- end }}
+
+{{/*
+Labels to use on serviceMonitor.spec.selector
+*/}}
+{{- define "thanos.servicemonitor.selector" -}}
+{{- include "thanos.servicemonitor.label" $ }}
```

Comment:
Is this calling a helper? I can't find where `thanos.servicemonitor.label` is defined, maybe you meant `thanos.servicemonitor.matchLabels`?